src/core/app.py

```diff
@@ -1,9 +1,14 @@
 import configparser
 import tkinter as tk
+import json
 from tkinter import ttk
 
 from core import generator
+from core import tooltip
 
+# current folder is `dist`
+with open('./../constants.json', 'r') as file:
+    constants = json.load(file)
```

Comment:
```suggestion
    tooltips_text = json.load(file)
```
Just updating this to match the updated file name. Going with `tooltips_text` to avoid confusion with the `tooltip` module. Don't forget to go through and update this reference throughout. Do a good old Find & Replace on this.

Reply:
Addressed, changed the file name as well.